### DISCUSSION TITLE
Can I install native dependencies (not JS)?

### DISCUSSION THREAD

**Discussion by lucasltv** [callstack/react-native-builder-bob#368] [Q&A]
Hello, first of all, thanks for this incredible tool!

I have one question. 

I need to install a native dependency but Android and iOS (not via npm).

Example: Can I install the Twilio SDK (android and iOS) inside respective folders (not example folder).

Like a wrapper for this native SDK's. 

Thank you!

**Comment by descorp** [comment]
You can use `.podspec` and `android/build.gradle` to add dependencies for each platform respectively.

**Reply by lucasltv** [reply]
OK!! Got it! Thank you a lot!

**Reply by HTCZeus** [reply]
Can u share how did you do it ?
